### PR TITLE
New version of consul agent

### PR DESCRIPTION
**Pull request by ozlevka-ericom**
Depends https://github.com/EricomSoftwareLtd/SB/pull/1732

### CODE REVIEW

File: Setup/docker-compose_dev.yml [hunk 1]
```diff
@@ -140,9 +143,12 @@ services:
         - "CONSUL_BIND_INTERFACE=eth0"
         - "NUMBER_OF_EXPECTED=3" # works for single and multi node
         - "LOGSTASH_FIELDS=service=consulserver"
+        - "DATA_DIR={{.Service.Name}}{{.Task.Slot}}" #single node don't use this for multinode
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: What should be done for multinode? just comment this line?

File: Setup/docker-compose_dev.yml [hunk 2]
```diff
@@ -140,9 +143,12 @@ services:
...
+    volumes:
+      - consul:/consul/data
 
   consul:
-      image: securebrowsing/shield-configuration:latest
+      image: securebrowsing/shield-consul-agent:latest
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: I agree the new name is better, 
what will happen on upgrade? i.e. a system is deployed with shield-configuration component and we deploy a new system with consul-agent
I assume this requires system restart
Please test that we will do the right thing

**Comment by ErezPasternak** [comment]
@benyh
We can’t add to current version yet

בתאריך יום א׳, 31 בדצמ׳ 2017 ב-9:51 מאת Beny H. <notifications@github.com>:

File: Setup/docker-compose_dev.yml (hunk 1 above)
Comment (ozlevka-ericom) [reply Setup/docker-compose_dev.yml]: Yes.
In multinode each volume on separate node

File: Setup/docker-compose_dev.yml (hunk 2 above)
Comment (ozlevka-ericom) [reply Setup/docker-compose_dev.yml]: I not think so...
Because this is new service... And all should start automatically with new service....

Comment (ozlevka-ericom) [reply Setup/docker-compose_dev.yml]: Will test this...

**Pull request merged**